dev/apollo-federation/supergraph-config.yaml

```diff
@@ -0,0 +1,6 @@
+federation_version: =2.3.2
+subgraphs:
+  main:
```

Comment:
```suggestion
  public:
```

Reply:
we want to converge to `public` instead of `main` I think.

Reply:
done https://github.com/GaloyMoney/galoy/pull/2940/commits/aeed158af97cbea43267dd6bb6a5ab123a0caaea